tools/base/test/define-element.test.ts

```diff
@@ -188,8 +188,8 @@ describe('define-element', function () {
                 this.skip();
             }
             const error = {
-                webkit: 'Cannot define multiple custom elements with the same tag name',
-                firefox: `'${name}' has already been defined`,
+                webkit: `"${name}" has already been used with this registry`,
```

Comment:
The [polyfill](https://github.com/webcomponents/polyfills/blob/master/packages/scoped-custom-element-registry/src/scoped-custom-element-registry.js) changes the error message on redefinations so we would need to keep it same for all three browser here.